full.async/src/full/async.clj

```diff
@@ -78,6 +104,7 @@
         ; doall to check for throwables right away
         (doall)))
 
+;; TODO lazy-seq vs. vector in <<! ?
```

Comment:
What do you mean? Lazy-seq is a big advantage here (which we leverage in our internal code for lazy loading various paged http resources). I guess it might be possible to make <<! return lazy-seq as well.